lib/ReactViews/Tour/TourExplanationBox.tsx

```diff
@@ -31,7 +31,7 @@ export const TourExplanationBox = styled(Box)`
   h1,
   h2,
   h3 {
-    margin-bottom: ${(p) => p.theme.spacing * 3}px;
+    margin-bottom: ${(p) => (p.theme.spacing as any) * 3}px;
```

Comment:
Can spacing be `as number` instead of any, or is that wrong?

Reply:
That will require casting to `unknown` first - eg `p.theme.spacing as unknown as number`
```
Conversion of type 'string' to type 'number' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```

Reply:
Yeah, probably better to leave it as any then, so it's easy to find in the future.